commons/src/main/java/org/dhis2/commons/dialogs/CustomDialog.java

```diff
@@ -10,16 +10,12 @@
 import android.view.View;
 import android.view.Window;
 
-import org.dhis2.R;
-import org.dhis2.databinding.CustomDialogBinding;
-import org.dhis2.utils.DialogClickListener;
-
-/**
- * QUADRAM. Created by frodriguez on 5/4/2018.
- */
+import org.dhis2.commons.R;
+import org.dhis2.commons.databinding.CustomDialogBinding;
 
 public class CustomDialog extends AlertDialog implements View.OnClickListener {
```

Comment:
I was thinking to not move java code to any module, and try to keep it 100% kotlin, what do you think? it's just a proposal :)